File: ejer-4.py
#aplanar un array en un nivel
arr = [1,2,[2,3,4], [12,[4,2],4]]
nuevo = []
for x in arr:
   if type(x) == int:
      nuevo.append(x)
   else:
      nuevo = [*nuevo, *x]

print(nuevo)

File: ejer-4.js
/*
Aplanar un array [[1,2], [12,2,[12,34,4]],[1,6,9]]
*/
const aplanar = (arr) => {
   return arr.reduce((acc, act) => acc.concat(act), [])
}
console.log(aplanar([2,[12,3,[4,2,1,2,2]], 2, 9]))